sdk/Makefile

```diff
@@ -48,7 +48,7 @@ Mgoogle/protobuf/field_mask.proto=github.com/gogo/protobuf/types:\
 		proto/*.proto; \
 	grep -v '^swagger:' ./../docs/proto/proto.md > ./../docs/proto/tmp-proto.md && mv ./../docs/proto/tmp-proto.md ./../docs/proto/proto.md
 	
-	go generate ./sdk/...
+	go generate ./...
```

Comment:
In my experience this change was required for the following to succeed: https://github.com/nginx/agent/blob/6a5bff8d3b529707ec9eed445ba43ec66aec4d28/Makefile#L104.

As this cmd references `./sdk/...` when the pwd is already inside `sdk/`.

I'd expect this works for others somehow. Open to alternate solutions.

Reply:
Yeah it's weird that it's worked before, I think your edit makes sense though. Also checked out to your branch and made sure the command worked ✅